cg/apps/cgstats/stats.py

```diff
@@ -105,28 +118,26 @@ def fastqs(self, flowcell: str, sample_obj: models.Sample) -> Iterator[Path]:
             files = self.root_dir.glob(pattern)
             yield from files
 
-    def document_path(self, flowcell_obj: models.Flowcell) -> str:
+    def document_path(self, flowcell_name: str) -> str:
         """Get the latest document path of a flowcell from supportparams"""
         query = (
             self.session.query(
                 models.Supportparams.document_path,
             )
             .join(models.Flowcell.demux, models.Demux.datasource, models.Datasource.supportparams)
-            .filter(models.Flowcell.flowcellname == flowcell_obj)
+            .filter(models.Flowcell.flowcellname == flowcell_name)
             .order_by(models.Supportparams.supportparams_id.desc())
             .first()
         )
-        document_path = query.document_path
-        return document_path
+        return query.document_path
```

Comment:
This kind of stuff will fail if first() returns None

Reply:
any suggestion? Perhaps it is ok to fail since it has been like that for ages